Mathlib/Topology/Compactness/CompactlyGeneratedSpace.lean

```diff
@@ -210,7 +211,7 @@ end UCompactlyGeneratedSpace
 
 section CompactlyGeneratedSpace
 
-variable {X : Type u} {Y : Type v} [TopologicalSpace X] [TopologicalSpace Y]
+variable {X : Type u} {Y : Type v} [tX : TopologicalSpace X] [tY : TopologicalSpace Y]
```

Comment:
why this change?

Reply:
I needed this for `theorem compactlyGeneratedSpace_of_coinduced` below, but I can do it locally.